### PR TITLE
Updates for Slashing and Operator Sets

### PR DESCRIPTION
**Pull request by wesfloyd**

### CODE REVIEW

File: docs/eigenlayer/operator-guides/operator-sets.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+* **Permissionless onboarding of AVSs:** Since slashing is localized to individual AVSs, there is no need for a common veto committee which means launching an AVS on EigenLayer remains permissionless.
+
+When Stakers deposit assets on EigenLayer, they are stored in accounting contracts known as Strategies. Strategies are different expressions of security on EigenLayer. Today, they represent different types of restaked assets (e.g., tokens) delegated to Operators that AVSs can leverage for securing their services and upholding cryptoeconomic guarantees. In order to make delegations slashable, Operators must allocate individual proportions of them as Unique Stake to Operator Sets. Allocations are exclusively slashable by the AVS that created that Operator Set.
+
+In figure 3 below, Operator 1 has a delegation of 100 staked ETH for the ETH Strategy. Operator 1 then allocates proportions of that ETH as Unique Stake in Operator Sets across several AVSs. 
+
+
+
+![Operator Allocations to Operator Sets](/img/operator-guides/operator-sets-figure-3.png)  
+***Figure 3: Operator Allocations to Operator Sets***
```
Comment (non-fungible-nelson) [inline docs/eigenlayer/operator-guides/operator-sets.md:43]: Let's remove the numbers

File: docs/eigenlayer/operator-guides/operator-sets.md [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+
+The 85 allocated ETH is slashable exclusively by the AVS originating each Operator Set. In this case, AVS 2, 3, and 4 can slash their associated Operator Sets 2, 3, and 4, respectively. 
+
+Let’s consider another example with three Operators. Figure 4 illustrates two Operator Sets instantiated by AVS 1. AVS 1 has created two Operator Sets for different tasks. For example, this AVS may use Operator Set 1 for assigning generation of ZK proofs to Operators, an expensive computation, and Operator Set 2 for verification of those proofs, a cheaper computation. 
+
+![Example of an AVS's Unique Stake](/img/operator-guides/operator-sets-figure-4.png)  
+***Figure 4: Example of an AVS’s Unique Stake***
```
Comment (non-fungible-nelson) [inline docs/eigenlayer/operator-guides/operator-sets.md:50]: And here

**Review by non-fungible-nelson — CHANGES_REQUESTED** [review]
LGTM with those changes

File: docs/eigenlayer/operator-guides/operator-sets.md (hunk 1 above)
Comment (wesfloyd) [reply docs/eigenlayer/operator-guides/operator-sets.md:43]: When you say "let's remove the numbers" ..  Which specific line numbers in the file would you like for me to remove? There are a lot of numbers, examples, and related diagrams in this section.

**Review by non-fungible-nelson — APPROVED** [review]

File: docs/eigenlayer/operator-guides/operator-sets.md [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+eigenlayer operator allocations show 
+	--network holesky 
+	--operator-address <operator-address> 
+	--strategy-addresses <comma-seperated-strategy-addresses>
+
+```
+
+Register and Deregister from Operator Set
+
+```
```
Comment (shrimalmadhur) [inline docs/eigenlayer/operator-guides/operator-sets.md:364]: can we make it not in one bash syntax - something like
Register to Operator set
```
eigenlayer operator register-operator-sets 
	--operator-address <operator-address> 
	--avs-address <avs-service-manager-address> 
	--operator-set-ids <comma-seperated-list-of-operator-set-ids>
	--caller-address <address-of-caller>
```

De-register from operator sets
```
eigenlayer operator deregister-operator-sets 
	--operator-address <operator-address> 
	--avs-address <avs-address> 
	--operator-set-ids <comma-seperated-list-of-operator-set-ids>
	--caller-address <address-of-caller>
```

also I added a flag in the command, so let's reflect that.

Comment (wesfloyd) [reply docs/eigenlayer/operator-guides/operator-sets.md:364]: done

**Pull request merged**